docs/reference/evercrypt/dh/ec/index.md

```diff
@@ -0,0 +1,38 @@
+# Elliptic-Curve
+
+Diffie-Hellman key exchange on Curve25519.
+
+## API Reference
+
+```C
+#include "EverCrypt_Curve25519.h"
+```
+
+```{doxygenfunction} EverCrypt_Curve25519_scalarmult
+```
+
+Compute the scalar multiple of a point.
+
+`shared` Pointer to 32 bytes of memory, allocated by the caller, where the resulting point is written to.
+`my_priv` Pointer to secret/private key.
+`their_pub` Pointer to public point.
+
+```{doxygenfunction} EverCrypt_Curve25519_secret_to_public
+```
+
+Calculate a public point from a secret/private key.
+
+This computes a scalar multiplication of the secret/private key with the curve's basepoint.
+
+`pub` Pointer to 32 bytes of memory, allocated by the caller, where the resulting point is written to.
```

Comment:
This is inconsistent now with the other argument descriptions that don't say anything about size or who's allocating. Please make sure we have the same level of detail for the different arguments in this PR (I think all the other ones are input, so maybe only add lengths?).